app/helpers/controllers/validateOrderItemFormBulk.js

```diff
@@ -36,7 +36,7 @@ export const validateOrderItemFormBulk = ({ data, selectedPrice, orderItemType }
         field: 'Price',
         id: 'PriceLessThanMax',
       });
-    } else if (parseFloat(data.price) > selectedPrice.price) {
+    } else if (parseFloat(data.price) > (selectedPrice.originalPrice || selectedPrice.price)) {
```

Comment:
Should this be `} else if (parseFloat(data.price) > selectedPrice.originalPrice) {`?

As we only want the validation error to occur if the entered price is greater than the list price.

Reply:
I just left in the existing logic in case it is use anywhere else an added the possibility of having an 'original price'.